src/mscorlib/src/System/Diagnostics/Eventing/TraceLogging/TraceLoggingDataCollector.cs

```diff
@@ -49,17 +49,6 @@ public void EndBufferedArray(int bookmark, int count)
             DataCollector.ThreadInstance.EndBufferedArray(bookmark, count);
         }
 
-        /// <summary>
-        /// Adds the start of a group to the event.
-        /// This has no effect on the event payload, but is provided to allow
-        /// WriteMetadata and WriteData implementations to have similar
-        /// sequences of calls, allowing for easier verification of correctness.
```

Comment:
removed method. I'm removing what you've flagged but it seems better to have a separate issue to remove dead stuff from eventing.